x-pack/plugins/lens/common/expressions/datatable/transpose_helpers.ts

```diff
@@ -118,7 +118,7 @@ function updateColumnArgs(
 ) {
   args.columns = [...bucketsColumnArgs];
   // add first column from each group, then add second column for each group, ...
-  transposedColumnGroups[0].forEach((_, index) => {
+  transposedColumnGroups[0]?.forEach((_, index) => {
```

Comment:
This is exactly why we enabled [`noUncheckedIndexedAccess`](https://www.typescriptlang.org/tsconfig#noUncheckedIndexedAccess) in https://github.com/elastic/elastic-charts/pull/2006

I don't know why typescript excluded this flag from `strict mode` when this is clearly an `undefined` union.

Reply:
We could consider adding this setting just for our code... it makes sense to me